skyportal/models.py

```diff
@@ -188,6 +203,13 @@ def desi_dr8_url(self):
                             secondary='join(Group, group_sources).join(group_users)',
                             primaryjoin='group_users.c.user_id == users.c.id')
 
+GroupSource.saved_as_source_by_id = sa.Column(sa.ForeignKey("users.id"), nullable=True)
```

Comment:
Is it correct to store the id here?  What if a source gets saved via API?

Reply:
I suppose if this is done via an API call, we'll have the handler specify the token's owner here (tokens are always associated with users).